practice_app/apps/article/migrations/0001_initial.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.5 on 2022-05-13 13:10
```

Comment:
The migrations folder is created by Django. This folder will be added to .gitignore on future commits. This folder is redundant, please remove it.

Reply:
This folder is also removed with the latest push.